changelogs/fragments/10299-github_app_access_token-lookup.yml

```diff
@@ -0,0 +1,4 @@
+bugfixes:
+  - github_app_access_token lookup plugin - Support both jwt and pyjwt to avoid conflict with other modules requirements (https://github.com/ansible-collections/community.general/issues/10299)
+deprecated_features:
+  - github_app_access_token lookup plugin - Support both jwt and pyjwt. jwt support is deprecated, and will be removed in community.general 14.0.0 (https://github.com/ansible-collections/community.general/issues/10299)
```

Comment:
Generally deprecations should only be added if the replacement feature has been there for a while (like: next major release). Can you please remove the deprecation from this PR? (You can create another PR to re-add it once this PR is merged, then we can merge that without backporting to `stable-11` so the deprecation will happen in 12.0.0.)
```suggestion
```